wagtail/admin/action_menu.py

```diff
@@ -44,11 +44,10 @@ def is_shown(self, context):
             'user_page_permissions' = a UserPagePermissionsProxy for the current user, to test permissions against
             may also contain:
             'user_page_permissions_tester' = a PagePermissionTester for the current user and page
+            'lock' = a Lock object if the page is locked, otherwise None
+            'locked_for_user' = True if the lock prevents the current user from editing the page
```

Comment:
Do we guarantee that the context will always contain these variables? If so, they should be listed before "may also contain". If not, the code below needs to guard against them being missing with `context.get(...)`.